packages/editor/src/components/blocks/radio/Radio.tsx

```diff
@@ -0,0 +1,88 @@
+import type { Prettify, Radio, AlignmentType } from '@axonivy/form-editor-protocol';
+import type { ComponentConfig, FieldOption, UiComponentProps } from '../../../types/config';
+import './Radio.css';
+import { baseComponentFields, defaultBaseComponent } from '../base';
+import IconSvg from './Radio.svg?react';
+import { Field, Label, RadioGroup, RadioGroupItem } from '@axonivy/ui-components';
+
+type RadioProps = Prettify<Radio>;
+
+const layoutOptions: FieldOption<AlignmentType>[] = [
+  { label: 'horizontal', value: 'lineDirection' },
+  { label: 'vertical', value: 'pageDirection' }
+] as const;
+
+export const defaultInputProps: Radio = {
+  label: 'Label',
+  layout: 'lineDirection',
```

Comment:
and why layout? I would call this property orientation...

Reply:
I called it layout because in PrimeFace, this property is also called layout (see PrintScreen above). However, I can also customise this in the core if you like. But I already use the label ‘Orientation’ for the property inside the FormEditor, as I also think this is a better name.

![grafik](https://github.com/user-attachments/assets/c105c3c5-37cd-4546-af9c-1061f3808723)